.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,35 @@
 <!---
+
 Thanks for creating a pull request!
 
-Please start your PR title with the appropriate prefix as specified by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#specification):
+Checklist:
 
-chore: [your title]
-docs: [your title]
-feat: [your title]
-fix: [your title]
-refactor: [your title]
-rfc: [your title]
-test: [your title]
--->
+ - Documentation in README, crate, module, function and/or other locations added/adapted.
 
+ - New tests added and/or existing tests adapted.
 
-## Intent
-<!---
-Please describe what you want to achieve here.
-Keep your PR limited to one logical change.
-For example, if you add a feature, don't include fixes you made along the way
-Please open another PR for more contributions.
--->
+ - PR is limited to a single logical change.
+   For example, if you add a feature, don't include fixes you made along the way
 
-## Description
-<!--- 
-For complex PRs, please describe what you did to achieve your goal here.
-Did you consider different approaches, where there any trade-offs you made?
-More information about why you implemented your solution the way you did helps reviewers.
-This section can be removed for obvious changes like typo fixes.
--->
+ - At least the first commit has a clear title and a descriptive message containing reasoning for the change.
+   This will be included in the final commit message after squashing and merging.
+
+ - PR title starts with an appropriate prefix as specified by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#specification):
+
+       chore: [your title]
+       docs: [your title]
+       feat: [your title]
+       fix: [your title]
+       refactor: [your title]
+       rfc: [your title]
+       test: [your title]
+
+ - PR description explains what it intends to achieve (linking to issues where appropriate).
+
+ - For more complex PRs, PR description explains _how_ this was achieved.
+   Did you consider different approaches, were there any trade-offs you made?
+   More information about why you implemented your solution the way you did helps reviewers.
+
+(Feel free to delete this comment once you've checked your PR against the requirements, for a draft PR it can be useful to copy not-yet-completed steps out to a TODO section so reviewers know the current state).
```

Comment:
Could you format the requirements as a TODO section to make copying more convenient?